app/src/main/res/layout/activity_interactive_feedback.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              style="@style/MessageActivityLayout">
+
+    <LinearLayout
+            android:orientation="vertical"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+        <TextView
+                android:text="@string/feedback_header"
+                android:id="@+id/header"
+                style="@style/MessageActivityText"/>
+        <LinearLayout
+                android:orientation="vertical"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="25dp">
+            <TextView
+                    android:text="@string/feedback_rating_question1"

```

Comment:
Hi Zita, 
Can you rename feedback_rating_question1 to feedback_rating_question

Reply:
Hello Benard, I changed it.

Reply:
Hi @rezita, Rename the string in the other languages string.xml

Reply:
Hello @benamonya , I did it.